script/OFTWrapper.s.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "utils/BaseScript.sol";
+import "mixins/OFTWrapper.sol";
+import "oracles/WitnetOracle.sol";
+
+contract OFTWrapperScript is BaseScript {
+    using DeployerFunctions for Deployer;
+    mapping (uint => uint) fixed_exchange_rate;
+
+    constructor() {
+       fixed_exchange_rate[1] = 539272521368673;
```

Comment:
I think it would be nice to use Create3 factory here to have the same address. See Precrime.s.sol